docs/asciidoc/modules/ROOT/partials/usage/apoc.util.compress.adoc

```diff
@@ -7,7 +7,7 @@ RETURN apoc.util.compress('Mätrix II 哈哈😄123', {charset: 'UTF-32'}) AS va
 [opts="header"]
 |===
 | value
-| [31, 139, 8, 0, 0, 0, 0, 0, 0, 255, 99, 96, 96, 240, 101, 96, 96, 120, 2, 196, 37, 64, 92, 4, 196, 153, 64, 92, 1, 196, 10, 64, 236, 9, 197, 64, 118, 200, 9, 48, 102, 252, 198, 2, 228, 27, 2, 177, 17, 16, 27, 3, 0, 113, 131, 223, 46, 64, 0, 0, 0]
+| {"0":31.0,"1":-117.0,"2":8.0,"3":0.0,"4":0.0,"5":0.0,"6":0.0,"7":0.0,"8":0.0,"9":-1.0,"10":99.0,"11":96.0,"12":96.0,"13":-16.0,"14":101.0,"15":96.0,"16":96.0,"17":120.0,"18":2.0,"19":-60.0,"20":37.0,"21":64.0,"22":92.0,"23":4.0,"24":-60.0,"25":-103.0,"26":64.0,"27":92.0,"28":1.0,"29":-60.0,"30":10.0,"31":64.0,"32":-20.0,"33":9.0,"34":-59.0,"35":64.0,"36":118.0,"37":-56.0,"38":9.0,"39":48.0,"40":102.0,"41":-4.0,"42":-58.0,"43":2.0,"44":-28.0,"45":27.0,"46":2.0,"47":-79.0,"48":17.0,"49":16.0,"50":27.0,"51":3.0,"52":0.0,"53":113.0,"54":-125.0,"55":-33.0,"56":46.0,"57":64.0,"58":0.0,"59":0.0,"60":0.0}
```

Comment:
do you know why the format changed here?

Reply:
Because the return value is no longer `List<Long>` but `byte[]`